service/kvmclusterconfig/v1/resource/encryptionkey/update.go

```diff
@@ -16,5 +17,25 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 // NewUpdatePatch computes appropriate Patch based on difference in current
 // state and desired state.
 func (r *Resource) NewUpdatePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*framework.Patch, error) {
-	return nil, nil
+	r.logger.LogCtx(ctx, "debug", "computing update patch for encryption key")
```

Comment:
I think I don't understand the meaning of logs here. I think earlier you instructed me to add them to Get* functions but here you say that they shouldn't be present in New* function. When testing I didn't notice framework logging anything. Is that because of logging configuration?

Reply:
It is just that we never had logs in `NewUpdatePatch` so far because no real logic happens here.